data.py-dist

```diff
@@ -20,18 +24,42 @@ HOSTS = {
 #    "testhost1": {"user": "root", "password": "", 'skip_xo_config': True},
 }
 
+NETWORKS = {
+#    "MGMT": "7a48464c-1e70-4e2e-a449-9f2514564381",
+}
+
 # PXE config server for automated XCP-ng installation
 PXE_CONFIG_SERVER = 'pxe'
 
 # Default VM images location
 DEF_VM_URL = 'http://pxe/images/'
 
+# Default shared ISO SR
+ISOSR_SRV = "nfs-server"
+ISOSR_PATH = "/srv/iso-sr"
+
+# Tools
+TOOLS = {
+#    "iso-remaster": "/home/user/src/xcpng/xcp/scripts/iso-remaster/iso-remaster.sh",
+}
+
 # Values can be either full URLs or only partial URLs that will be automatically appended to DEF_VM_URL
 VM_IMAGES = {
     'mini-linux-x86_64-bios': 'alpine-minimal-3.12.0.xva',
     'mini-linux-x86_64-uefi': 'alpine-uefi-minimal-3.12.0.xva'
 }
 
+# FIXME: use URLs and an optional cache?
+ISO_IMAGES = {
+#    'xcpng-8.0': "/home/user/iso/xcp-ng-8.0.0.iso",
+#    'xcpng-8.1': "/home/user/iso/xcp-ng-8.1.0-2.iso",
+#    'xcpng-8.2.0': "/home/user/iso/xcp-ng-8.2.0.iso",
+#    'xcpng-8.2.1': "/home/user/iso/xcp-ng-8.2.1.iso",
+#    'xcpng-8.2.1-2023': "/home/user/iso/xcp-ng-8.2.1-20231130.iso",
+#    'xcpng-8.3-beta2': "/home/user/iso/xcp-ng-8.3.0-beta2.iso",
```

Comment:
maybe `xcpng-8.3.0-beta2` ? because we will have at some point an `xcpng-8.3.1-beta2`... It looks like a bit more consistent IMHO.